docs/modules/ROOT/pages/cops_rspec.adoc

```diff
@@ -4184,6 +4184,59 @@ expect(foo.something?).to be_truthy
 * https://rspec.rubystyle.guide/#predicate-matchers
 * https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/PredicateMatcher
 
+== RSpec/ProperlyItSpecify
+
+|===
+| Enabled by default | Safe | Supports autocorrection | Version Added | Version Changed
+
+| Pending
+| Yes
+| Yes
+| <<next>>
+| -
+|===
+
+Check for `specify` and `it` are used properly.
+
+Use specify if the example doesn't have a description,
+use it for examples with descriptions.
+
+=== Examples
+
+[source,ruby]
+----
+# bad
+it do
+  # ...
+end
+
+# good
+specify do
+  # ...
+end
+
+# bad
+specify 'it sends an email' do
+  # ...
+end
+
+# good
+it 'sends an email' do
+  # ...
+end
+
+# bad
+specify { is_expected.to be_truthy }
```

Comment:
What about `specify { expect(sqrt(4)).to eq(2) }`?

Reply:
https://rspec.rubystyle.guide/#it-and-specify
> An exception is one-line example, where it is preferable.

Maybe, since it is described above, I think it would be better to consider it as bad code.

Reply:
I would say that for one-liners, `it` is preferred when `is_expected` is used. Other one-liners, like the one @pirj mentioned probably `specify` is better.

Reply:
Indeed it is. I updated this PR.